lohica-js-network/src/components/loginForm/LoginEmailComponent.js

```diff
@@ -12,9 +12,9 @@ class LoginEmailComponent extends React.Component {
     constructor(props){
         super(props);
 
-        this.state = {
-            show: false
-        }
+        // this.state = {
```

Comment:
Remove all the comments from the PR if you do not need it.

Reply:
done